examples/fetch/index.tsx

```diff
@@ -23,11 +23,55 @@ const App = () => {
     }
   };
 
+  async function streamToArrayBuffer(stream: any) {
+    const reader = stream.getReader();
+    while (true) {
+      console.log("await read");
+      const { done, value } = await reader.read();
+      console.log("done", done);
+      if (done) {
+        break;
+      } else {
+        console.log(value.byteLength);
+        // @ts-ignore
+        const text = globalThis.TextCodecHelper.decode(value);
+        setData(text);
+        const currentTime = new Date();
+        console.log(text);
+        // @ts-ignore
+        console.log("duration ", currentTime.getTime() - globalThis.startTime.getTime());
+      }
+    }
+  }
+
+  const handleButtonClickStreaming = async () => {
+    setError(null);
+
+    try {
+      console.log("fetch streaming");
+      // @ts-ignore
+      globalThis.startTime = new Date();
```

Comment:
I don't think seting `startTime` on `globalThis` is a good practice. Try to use a local variable or a state instead.